docs/4.1. Behaviour - Registration.md

```diff
@@ -10,106 +10,107 @@ This document covers expected client and server behaviour for interactions betwe
 
 #### Resilient Node Behaviour
 
-In some deployments, it may be desirable to send multiple copies of essence (video/audio/data) via redundant paths in order to provide guarantees about availability. In these situations, it is likely that a single Node will be used at the point of content creation and transmission via the two paths, and again a single Node used for reception from the two paths.
+In some deployments, multiple copies of essence (video/audio/data) are sent via redundant paths in order to provide guarantees about availability. In these situations, it is likely that a single Node will be used at the point of content creation and transmission via the two paths, and again a single Node used for reception from the two paths.
 
-The data model employed by NMOS allows for this situation by modelling a single Flow sent via two independent Senders which are assigned to separate physical network interfaces on a Node.
+One way NMOS can model this is as a single Flow sent via two independent Senders which are assigned to separate physical network interfaces on a Node.
 
-This mode of operation is applicable to both the 'Peer to Peer' and 'Registered' discovery case. It is also envisaged in the Registered case that operation may be desirable with a single registry, or with two which operate totally independently on the two different networks.
+Resilient operation via redundancy is also applicable to the NMOS APIs and both 'peer-to-peer' and 'registered' discovery. Registered discovery MAY use either a single registry, or an independent registry for each network.
 
-In order to support this mode and ensure persistence in the registry upon link failure, a Node operating in this mode may register with Registration APIs (and host its Node API) via multiple independent network interfaces. In the case where separate registries are deployed for the multiple network links the Node must ensure it registers the correct 'href' with each registry (see [Node API](../APIs/NodeAPI.raml) /self resource). From v1.1, the 'api' attribute of the Node /self resource adds the capability to list multiple endpoints for the same Node API, removing the requirement for this 'href' specialisation.
+In order to support this mode and ensure persistence in the registry upon link failure, a Node operating in this mode MAY register with Registration APIs (and host its Node API) via multiple independent network interfaces. If separate registries are deployed for the multiple network links the Node MUST ensure it registers the correct `href` with each registry (see [Node API](../APIs/NodeAPI.raml) `/self` resource). From v1.1, the `api` attribute of the Node `/self` resource adds the capability to list multiple endpoints for the same Node API, removing the requirement for this `href` specialisation.
```

Comment:
Andrew's suggestion can't be applied because GitHub says it's outdated, so here it is again:

```suggestion
In order to support this mode and ensure persistence in the registry upon link failure, a Node operating in this mode MAY register with Registration APIs (and host its Node API) via multiple independent network interfaces.

Clients are encouraged to use the `api` attribute rather than the deprecated `href` attribute when accessing a Node API (see [Node API](../APIs/NodeAPI.raml) `/self` resource) as the `href` attribute is likely to target a single network interface only. Nodes which need to support this mode of operation at v1.0 MUST register a different `href` for the same Node `id` with each registry, even though this is a workaround which technically breaches data model constraints.
```